doc/lua_api.txt

```diff
@@ -3758,7 +3758,9 @@ It can be created via `Settings(filename)`.
 
 #### Methods
 * `get(key)`: returns a value
-* `get_bool(key)`: returns a boolean
+* `get_bool(key, default)`: returns a boolean
+    * `default` is the value returned if `key` is not found.
+    * If `default` is not set and `key` is not found, then `nil` is returned.
```

Comment:
`Returns ``nil`` if ``key`` is not found and ``default`` not specified.`
To start the sentence with the relevant information a player would look for.

Reply:
Okay.